pallets/collator-selection/src/lib.rs

```diff
@@ -127,6 +130,12 @@ pub mod pallet {
 		/// This does not take into account the invulnerables.
 		type MaxCandidates: Get<u32>;
 
+		/// Minimum number of candidates that we should have. This is used for disaster recovery.
+		///
+		/// This does not take into account the invulnerables.
+		type MinCandidates: Get<usize>;
```

Comment:
```suggestion
		type MinCandidates: Get<u32>;
```

Should keep this as u32 for best practice